src/stores/Frame/Region/RegionStore.ts

```diff
@@ -481,11 +484,23 @@ export class RegionStore {
         }
     };
 
+    /**
+     * Sets the size for regions and annotations
+     *
+     * @param p - Specifies the x and y size components.
+     *            For line region and annotation, vector annotation, and ruler annotation, the function sets the individual new x and y components of the linear segment.
```

Comment:
Perhaps revise it to "For line regions and annotations, vector annotations, and ruler annotations, the function sets the new start and end positions while keeping the rotation within the same quadrant."